rasa_core/tracker_store.py

```diff
@@ -30,7 +30,14 @@ def __init__(self, domain, event_broker=None):
         # type: (Optional[Domain], Optional[EventChannel]) -> None
         self.domain = domain
         self.event_broker = event_broker
-
+    def find_tracker_store(self, store=None):
+        if store is None:
+            return InMemoryTrackerStore(self.domain)
+        elif store.store_type == 'redis':
+            return RedisTrackerStore(domain=self.domain,host=store.url,db=store.db,password=store.password,record_exp=store.timeout)
```

Comment:
It would be great if the user could also define a certain port through the configuration file.
Hence, we should probably change this line to:
```python
return RedisTrackerStore(domain=self.domain, 
                         host=store.url, 
                         port=store.port,
                         db=store.db, 
                         password=store.password, 
                         record_exp=store.timeout)

```